File: AtlasWeb/Core/Modules/Tibbi_Surec/Manipulasyon_Modulu/ManipulationProcedureRequestInfoViewModel.ts
//$515BBB73
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ManipulationProcedure } from "NebulaClient/Model/AtlasClientModel";
import { Type } from "NebulaClient/ClassTransformer";

export class ManipulationProcedureRequestInfoViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ManipulationProcedureRequestInfoViewModel, Core';
    @Type(() => ManipulationProcedure)
    public _ManipulationProcedure: ManipulationProcedure = new ManipulationProcedure();
    public ProcedureName: string;
    public Description: string;
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Manipulasyon_Modulu/ManipulationProcedureRequestInfoViewModel.cs
//$07E0202E
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;

namespace Core.Controllers
{
    public partial class ManipulationProcedureServiceController
    {
    }
}

namespace Core.Models
{
    public partial class ManipulationProcedureRequestInfoViewModel : BaseViewModel
    {
        public string ProcedureName
        {
            get;
            set;
        }

        public string Description
        {
            get;
            set;
        }
    }
}
